proto/content_analysis/sdk/acknowledgement.proto

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 The Chromium Authors.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto2";
+
+option optimize_for = LITE_RUNTIME;
+
+package content_analysis.sdk;
+
+message Acknowledgement {
+  enum Status {
+    STATUS_UNKNOWN = 0;
+    SUCCESS = 1;
+    FAILURE = 2;
+  }
+  required Status status = 1;
+}
```

Comment:
I setup Acknowledgement and Handshake as enums directly, but then it asks to change their member names to prevent naming collisions.

```
  content_analysis/sdk/analysis.proto:22:3: "STATUS_UNKNOWN" is already defined in "content_analysis.sdk".
  content_analysis/sdk/analysis.proto:22:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "STATUS_UNKNOWN" must be unique within "content_analysis.sdk", not jus
  t within "Acknowledgement".
  content_analysis/sdk/analysis.proto:23:3: "SUCCESS" is already defined in "content_analysis.sdk".
  content_analysis/sdk/analysis.proto:23:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "SUCCESS" must be unique within "content_analysis.sdk", not just withi
  n "Acknowledgement".
  content_analysis/sdk/analysis.proto:24:3: "FAILURE" is already defined in "content_analysis.sdk".
  content_analysis/sdk/analysis.proto:24:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "FAILURE" must be unique within "content_analysis.sdk", not just within "Acknowledgement"

Reply:
Would it be better to have them
1/ both as ENUMs, but change the names of their values
2/ both as messages with status (leave as-is)
3/ one as ENUM, one as message.

Reply:
Am keeping them as messages as it keeps the namespace cleaner.

Reply:
Ack.  Espeically when seeing the next PR.